client/lib/wp/sync-handler/cache-invalidation.js

```diff
@@ -0,0 +1,101 @@
+/**
+ * Module dependencies
+ */
+import moment from 'moment';
+import { getLocalForage } from 'lib/localforage';
+import chrono from 'chrono-node';
```

Comment:
I don't think we should incur the overhead of a natural language parser just to set a parameter input. We may not even need multiple values. Instead I think we should just export some duration constants that can be used when calling the cache invalidator.